.circleci/config.yml

```diff
@@ -1,7 +1,9 @@
+version: 2.1
+
 defaults: &defaults
   working_directory: /tmp/material-ui
   docker:
-    - image: circleci/node:8.15
+    - image: circleci/node:8.15-browsers
```

Comment:
What do you think of reverting this change? The tests relying on Chrome Headless are flaky now:
- Tests real browsers https://circleci.com/gh/mui-org/material-ui/75600?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link
- Test umd release https://circleci.com/gh/mui-org/material-ui/75614?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link

cc @joshwooding

Reply:
Tests are flaky on 8.15 too: https://circleci.com/gh/mui-org/material-ui/76445. Need to pin to exact versions it seems.